src/Aspire.Hosting.SqlServer/SqlServerContainerImageTags.cs

```diff
@@ -11,6 +11,6 @@ internal static class SqlServerContainerImageTags
     /// <remarks>mssql/server</remarks>
     public const string Image = "mssql/server";
 
-    /// <remarks>2022-latest</remarks>
-    public const string Tag = "2022-latest";
+    /// <remarks>2025-latest</remarks>
+    public const string Tag = "2025-latest";
```

Comment:
@joperezr @eerhardt This has been problematic as it no longer works on Mac and we've had several complaints about this. I guess we'll need to more carefully handle major upgrades of container image tags.

Reply:
Sounds like we have some gap in our coverage, does this mean we should be running some tests in mac that we aren't? I'm not sure if constraining image tag adoption is something we should do. I much rather have more test coverage so we can detect when bumping versions causes issues so we prevent us finding out about these breaks when customers try it.

Reply:
I don't think we test any docker tests on macOS:

https://github.com/dotnet/aspire/blob/46c25293c14d8231fa1db59d9ce4e62e77471621/tests/Aspire.TestUtilities/RequiresDockerAttribute.cs#L22-L23

cc @radical